com.unity.render-pipelines.core/Editor/PostProcess/SRPLensFlareOverrideEditor.cs

```diff
@@ -0,0 +1,127 @@
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+namespace UnityEditor.Rendering
+{
+    /// <summary>
+    /// Editor for SRPLensFlareOverride: Lens Flare Data-Driven which can but added on any GameObject
+    /// </summary>
+    [CanEditMultipleObjects]
+    [CustomEditor(typeof(SRPLensFlareOverride))]
+    public class SRPLensFlareOverrideEditor : Editor
+    {
+        SerializedProperty m_LensFlareData;
+        SerializedProperty m_Intensity;
+        SerializedProperty m_Scale;
+        SerializedProperty m_MaxAttenuationDistance;
+        SerializedProperty m_MaxAttenuationScale;
+        SerializedProperty m_DistanceAttenuationCurve;
+        SerializedProperty m_ScaleByDistanceCurve;
+        SerializedProperty m_AttenuationByLightShape;
+        SerializedProperty m_RadialScreenAttenuationCurve;
+        SerializedProperty m_UseOcclusion;
+        SerializedProperty m_OcclusionRadius;
+        SerializedProperty m_SamplesCount;
+        SerializedProperty m_OcclusionOffset;
+        SerializedProperty m_AllowOffScreen;
```

Comment:
Here you do it right: it is cached into the Editor. So you compute it once in OnEnable() and then only use it in OnInspectorGUI(). You should do the same than here for above comments.